internal/output/output.go

```diff
@@ -38,7 +39,12 @@ func Print(c Config, t string, v interface{}) error {
 		if err != nil {
 			return err
 		}
-		return tmpl.Execute(os.Stdout, v)
+		w := tabwriter.NewWriter(os.Stdout, 0, 8, 2, '\t', 0)
```

Comment:
[np] The collection of numbers on this call really isn't very self documenting unless you dig into the method signatures. A comment as to the intent would be good here I think.

Reply:
will do